PubNub/Data/Service Objects/PNResult.m

```diff
@@ -83,22 +83,29 @@ - (id)copyWithZone:(NSZone *)zone {
     result.authKey = self.authKey;
     result.origin = self.origin;
     result.clientRequest = self.clientRequest;
-    result.serviceData = self.serviceData;
-    
+    [result updateData:self.serviceData];
+
     return result;
 }
 
 - (instancetype)copyWithMutatedData:(id)data {
     
     PNResult *result = [self copy];
-    result->_serviceData = [data copy];
-    
+    [result updateData:data];
+
     return result;
 }
 
++ (NSDictionary *)normalizeServiceData:(id)serviceData {
+    if (serviceData && ![serviceData isKindOfClass:[NSDictionary class]]) {
+        return @{@"information": serviceData};
```

Comment:
That's exactly what's going on. Unfortunately, PubNub just types `@"information"` everywhere instead of making a const. Should I change their code or just leave it go?